### DISCUSSION TITLE
[Q&A] Undocumented container image updates which do not follow the documented versioning scheme

### DISCUSSION THREAD

**Discussion by cospeedster** [fluent/fluentd#4639] [Q&A] [answered]
### What is a problem?

We recently implemented [Renovate](https://github.com/renovatebot/renovate) to manage our updates.

On September 10th, we updated to `fluent/fluentd:v1.17.1-1.0` with digest `sha256:40ef8cb68fc4b3689fe775658d83dd9f26588c7aece64f99f835c83765adb7c3`. Since then, we have received three digest updates for the same image tag, indicating changes to the image behind the tag:

- `sha256:f79bd36cd9b129dde6574a03c90f805b744901d30685e5dc6efa8dc97080cc86`
- `sha256:8341ba9935670c6fdf25d52151fce77c99272326f75c05e5a1a71bd7afa192ed`
- `sha256:39f225e5b0315aa9bb9a79c724c74a03fe48f5f77c57c21623b41b210908d969` (today)

According to your [README](https://github.com/fluent/fluentd-docker-image?tab=readme-ov-file#vxyz-ab), we understood that the **B** in your versioning scheme `vX.Y.Z-A.B` increases for minor image updates. However, it appears this is not the case.

Could you please provide your thoughts on this and share the changelog for these updates?

### Describe the configuration of Fluentd

unnecessary

### Describe the logs of Fluentd

unnecessary

### Environment

```markdown
- Fluentd version: v1.17.1
- TD Agent version: unnecessary
- Fluent Package version: unnecessary
- Docker image (tag): v1.17.1-1.0
- Operating system: unnecessary
- Kernel version: unnecessary
```

**Comment by kenhys** [comment, marked as answer]
Thank you for feedback.

It seems that the side effect of fluent/fluentd autobuild was enabled unexpectedly.

Now disabled that feature on hub.docker.com.
